### ISSUE TITLE
custom.css

### ISSUE THREAD

**Issue by quantransform** [jupyter/nbconvert#795]
I first use the following on commend line and get 404 for custom.css.
> jupyter nbconvert test.ipynb --to slides --post serve
WARNING:404 GET /custom.css (127.0.0.1) 

I then added a file called custom.css on the folder that I tried to --to slides. It worked as custom.css is now included. However, when I started to change custom.css, it won't pick it up and it continues to use the first one. I assume that it is put somewhere and won't be replaced? Any idea how I can solve this please?

Thank you.

**Comment by takluyver** [comment]
I don't think we put it anywhere, but it might be cached by your browser. Try hitting Ctrl-F5 a few times to refresh it.

**Comment by wefatherley** [comment]
Hi @quantransform , you might try `touch /path/to/custom.css` to update the file metadata after you make edits to it.
